docs/ModularizationLearningJourney.md

```diff
@@ -159,13 +159,21 @@ Using the above modularization strategy, the Now in Android app has the followin
    <td><code>TopicsRepository</code><br>
    </td>
   </tr>
+  <tr>
+   <td><code>core:designsystem</code>
+   </td>
+   <td>Design system, tokens and icons.
+   </td>
+   <td><code>NiaTheme</code><br>
+   <code>NiaIcons</code>
+   </td>
+  </tr>
   <tr>
    <td><code>core:ui</code>
    </td>
-   <td>UI components, composables and resources, such as icons, used by different features.
+   <td>UI components, composables and resources, used by different features.
```

Comment:
```suggestion
   <td>Composite UI components and resources used by feature modules, such as the news feed. Unlike the `designsystem` module, it is dependent on the data layer since it renders models, like news resources. 
```